phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         syntaxCheck="true">
+    <testsuites>
+        <testsuite name="unit">
+            <directory suffix="Test.php">./Tests</directory>
```

Comment:
ups, sorry. did just go through the notification emails. i will adapt and keep it in the symfony style.